ChangeLog

```diff
@@ -14,6 +24,7 @@
 	* man/BermudanSwaption.Rmd: Modified to reflect DiscountCurve
 	option and example updated
 	
+>>>>>>> eddelbuettel/master
```

Comment:
Whether plausible or not, it¹s what I did (see github desktop screenshot).

From:  Dirk Eddelbuettel <notifications@github.com>
Reply-To:  eddelbuettel/rquantlib
<reply+007042a2a6a294360d9fecfd27625ec3b84ff02bde5713c192cf00000001134dc7519
2a170ce03c49451@reply.github.com>
Date:  Friday, May 13, 2016 at 12:48 PM
To:  eddelbuettel/rquantlib <rquantlib@noreply.github.com>
Cc:  terry leitch <tleitch@uchicago.edu>, Author <author@noreply.github.com>
Subject:  Re: [eddelbuettel/rquantlib] Extend curve instruments to 100 years
(#47)


In ChangeLog 
<https://github.com/eddelbuettel/rquantlib/pull/47#discussion_r63214673> :
> @@ -14,6 +24,7 @@
>  	* man/BermudanSwaption.Rmd: Modified to reflect DiscountCurve
>  	option and example updated
>  	
> +>>>>>>> eddelbuettel/master
Not really a plausible story as those files weren't part of master.

This really does not have to be complicated, and I just explained at work
where we migrate some repos from SVN:
* start with a clean slate, ie no repos on your end, local or remote
* fork a remote repository; we consider master to be 'good' and the standard
(fork locally or remote? I have a remote fork of master)
* you now have a remote copy of master
* clone your remote; you now have a local copy   (locally, right, which I
have via a project open in Studio)
* maybe run R CMD build ...; R CMD check ... to validate tools
* create a branch:  git checkout -b feature/some-new-stuff   (have not been
doing this, working on clone directly in Studio)
* do work, run checks, ensure everything works, commit this and push it
(been pushing to my fork at github but seems you mean I need to keep local)
* your remote should check this on Travis  (not sure how I do this)
* once happy, form a pull request which will show you again which files are
involved  (I see files but do not see how to remove unnecessary files from
pull on github)
* it also shows you what the diff is (I see this)
* look it over; if you are really sure it makes then submit it
That should could down on the amount of noise going back and forth.


You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/eddelbuettel/rquantlib/pull/47/files/8704b4c2e29fe228f1d
6f2d3e846aa6311f01c4b#r63214673>